index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Week Planner</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body>
+    <h1>Weekly Planner</h1>
+    <section class="week"></section>
+    <section class="add">
+      <p>Add Entry</p>
+    </section>
```

Comment:
a section element should be used to separate your page into sections, it doesnt really make sense to do it here

You want to use a div or a button for your add entry button